easybuild/easyconfigs/t/ToFu/ToFu-1.4.0-CC-by-env.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/setup.py b/setup.py
```

Comment:
Please add description and author info to the patches.
And update checksums accordingly.

Reply:
@Louwrensth ping - any update on this?

Reply:
Thanks for the bump. Will do.

Reply:
Done.

Reply:
@verdurin, @akesandgren : should I do anything else? the PR still says it's waiting for some request...

Reply:
Doing a force push destroys the history and confuses travis, that's likely why it thinks there are unresolved review requests.

And, unless I'm blind, ToFu-1.4.0-foss-2018a-Python-2.7.14.eb is missing checksums for one patch and so is ToFu-1.4.0-intel-2018a-Python-2.7.14.eb

And the checksum for ToFu-1.4.0-no-git.patch is wrong for all easyconfigs.

Reply:
I recommend eb *.eb --inject-checksum --force